src/com/nutrons/nu17/RobotMap.java

```diff
@@ -9,12 +9,12 @@
 	public static final int BACK_RIGHT = 3;
 	
 	// Ports of intake
-	public static final int ROLLER_A = 0;
-	public static final int ROLLER_B = 0;
+	public static final int ROLLER_FIRST = 0;
```

Comment:
This is just as bad.  I should be able to read the variable and point to the bit of the robot it attaches to.